faq/faq.md

```diff
@@ -0,0 +1,65 @@
+<!--
+\-\--layout: page title: FAQ permalink: /faq/ order: 4 feature-img:
+\"examples/img/hero-bg.jpg\" feat_img_size: small \# Copyright (c) 2018,
+Intel Corporation. \# Licensed under Creative Commons Attribution 4.0
+International License \# <https://creativecommons.org/licenses/by/4.0/>
+\-\--
+-->
+
+# Hyperledger Sawtooth FAQ 
+
+---
+
+<!--
+========================
+-->
+
+Answers to frequently asked questions for Hyperledger Sawtooth. Many
+questions and answers originated from the \#sawtooth Rocket.Chat channel
+at <https://chat.hyperledger.org/channel/sawtooth>
+
+---
+
+## Sawtooth FAQ Sections
+
+-   [Hyperledger Sawtooth in General](sawtooth)
+-   [Installation and Configuration](installation)
+-   [Transaction Processing](transaction-processing)
+-   [Validator](validator)
+-   [Consensus Algorithms (including PoET)](consensus)
+-   [Client](client)
+-   [REST API](rest)
+-   [Permissioning](permissioning)
+-   [Using Docker](docker)
+-   [Upgrading Hyperledger Sawtooth](upgrade)
+
+---
+
+## FAQ Appendixes
+
+-   [Sawtooth Glossary](glossary)
+-   [Transaction Family Prefixes](prefixes)
+-   [Configuration Settings](settings)
+-   [Sawtooth Videos](videos)
+
+---
+
+## Contributing to This FAQ
+
+This FAQ is licensed under Creative Commons Attribution 4.0
+International License. You may obtain a copy of the license at:
+<http://creativecommons.org/licenses/by/4.0/>.
+
+We accept contributions via [GitHub](https://github.com/hyperledger/sawtooth-website) pull requests.
+Each commit must include a `Signed-off-by:` in the commit message <br/>
+(`git commit -s`). This sign-off means you agree the commit satisfies
+the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).\]
+
+<!--
+::: {.mininav}
+[NEXT](/faq/sawtooth/)
+:::
+-->
+
+© Copyright 2018, Intel Corporation.
```

Comment:
Remove all copyrights at the end of the files (this should now be in the comment on the top of the files).